src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/TemporaryAbsencesResource.kt

```diff
@@ -126,15 +126,15 @@ class TemporaryAbsencesResource(
   )
   @GetMapping(
     path = [
-      "/prison/{prisonId}/temporary-absences/{prisonNumber}"
+      "/prison/{prisonId}/temporary-absences/{prisonNumber}", "/temporary-absences/{prisonNumber}"
     ]
   )
   fun getTemporaryAbsence(
-    @Schema(description = "Prison ID", example = "MDI", required = true)
-    @PathVariable prisonId: String,
+    @Schema(description = "Prison ID", example = "MDI", required = false)
+    @PathVariable prisonId: String?,
```

Comment:
Is this for backwards compatibility? I.e. we plan to remove `"/prison/{prisonId}/temporary-absences/{prisonNumber}"` once the UI has been updated - or something along those lines?

Reply:
Yes exactly - for backwards compatibility until the frontend can be updated, then will go back and remove the old endpoint and prisonId :)